src/Compilers/Shared/BuildServerConnection.cs

```diff
@@ -454,6 +454,13 @@ private static bool CheckPipeConnectionOwnership(NamedPipeClientStream pipeStrea
             {
                 if (PlatformInformation.IsWindows)
                 {
+                    if (CoreClrShim.IsRunningOnCoreClr)
+                    {
+                        // CoreCLR doesn't have these APIs, so we just have to rely on
+                        // Windows ACLs for security
+                        return true;
```

Comment:
Unfortunately at this point on CoreClr we don't have ACLs 

https://github.com/dotnet/corefx/issues/24040

This means we no longer have ACLs or identity verification.